public/assets/scss/_faq.scss

```diff
@@ -0,0 +1,232 @@
+.faq-holding {
+    background-image: url('../img/backgrounds/faq-holding.png');
+    background-size: 100% 100%;
+    background-repeat: no-repeat;
+    justify-content: center;
+    height: 35rem;
+    width: 100%;
+
+    &-heading {
+        color: $blue;
+        text-align: center;
+        padding-top: 10rem;
+        padding-bottom: 2rem;
+    }
+    &-text {
+        text-align: center;
+        @extend %heading-text-font-stack;
+        padding-bottom: 3rem;
+    }
+    &-form {
+        text-align: center;
+        align-content: center;
+        margin: auto;
+        background-color: $white;
+        padding: 1rem 0.5rem;
+        border-radius: 0.5rem;
+        box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
```

Comment:
The box-shadow used in the Figma file is `box-shadow: 0px 15px 30px rgba(196, 196, 196, 0.3);`